project/Versions.scala

```diff
@@ -25,14 +38,22 @@ object Versions {
   val JDK             = "1.7"
   val JodaC           = "1.2"
   val JodaT           = "2.3"
-  val JOpt            = "3.2"//4.7
-  val Kafka           = "0.8.0"//https://github.com/apache/spark/pull/3631
+  val JOpt            = "3.2"
+  val Kafka           = "0.8.2.0"
+  val Kafka210        = "0.8.1.1"
   val Lzf             = "0.8.4"
   val CodaHaleMetrics = "3.0.2"
-  val Scala           = "2.10.4"
   val ScalaTest       = "2.2.2"
   val Scalactic       = "2.2.2"
-  val Slf4j           = "1.7.7"
+  val Slf4j           = "1.6.1"//1.7.7"
```

Comment:
@jacek-lewandowski this is it:

[error] (spark-cassandra-connector/*:update) java.lang.IllegalStateException: impossible to get artifacts when data has not been loaded. IvyNode = org.slf4j#slf4j-api;1.6.1

We have one dep that excludes slf4j and that is kafka - even if you remove that exclusion I see the same error.